[441.844701767] 2019-01-03 10:51:06,175 - INFO - launcher: Starting sanity checks
[442.844390869] 2019-01-03 10:51:06,176 - INFO - launcher: Checking for Python 3.5+
[443.845033646] 2019-01-03 10:51:06,177 - INFO - launcher: Checking console encoding
[443.845033646] 2019-01-03 10:51:06,177 - INFO - launcher: Setting console encoding to UTF-8
[443.845033646] 2019-01-03 10:51:06,177 - INFO - launcher: Ensuring we're in the right environment
[447.841882706] 2019-01-03 10:51:06,181 - INFO - launcher: Adding local bins/ folder to path
[6995.595216751] 2019-01-03 10:51:12,729 - INFO - launcher: Required checks passed.
[6996.595382690] 2019-01-03 10:51:12,730 - INFO - launcher: Optional checks passed.
[6997.597455978] 2019-01-03 10:51:12,731 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[21.5927863121032715] 2019-01-03 10:51:27,326 - WARNING - musicbot.config | In config.py::MainThread(11756), line 115 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[21.6109035015106201] 2019-01-03 10:51:27,344 - INFO - musicbot.config | In config.py::MainThread(11756), line 126 in run_checks: Using i18n: config/i18n/en.json
[21.6258876323699951] 2019-01-03 10:51:27,359 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 219 in _setup_logging: Set logging level to INFO
[21.6298861503601074] 2019-01-03 10:51:27,363 - DEBUG - musicbot.json | In json.py::MainThread(11756), line 8 in __init__: Init JSON obj with config/i18n/en.json
[22.4102017879486084] 2019-01-03 10:51:28,144 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 84 in __init__: Starting MusicBot 1.9.8_3
[22.4111850261688232] 2019-01-03 10:51:28,145 - WARNING - musicbot.bot | In bot.py::MainThread(11756), line 87 in __init__: Autoplaylist is empty, disabling.
[25.2025589942932129] 2019-01-03 10:51:30,936 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 915 in on_ready: Connection established, ready to go.
[25.2035574913024902] 2019-01-03 10:51:30,937 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 333 in _cache_app_info: Caching app info
[25.3295581340789795] 2019-01-03 10:51:31,063 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 737 in _scheck_ensure_env: Ensuring data folders exist
[25.3425724506378174] 2019-01-03 10:51:31,076 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 753 in _scheck_server_permissions: Checking server permissions
[25.3425724506378174] 2019-01-03 10:51:31,076 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 757 in _scheck_autoplaylist: Auditing autoplaylist
[25.3435580730438232] 2019-01-03 10:51:31,077 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 761 in _scheck_configs: Validating config
[25.3435580730438232] 2019-01-03 10:51:31,077 - DEBUG - musicbot.config | In config.py::MainThread(11756), line 216 in async_validate: Validating options...
[25.3435580730438232] 2019-01-03 10:51:31,077 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 764 in _scheck_configs: Validating permissions config
[25.3435580730438232] 2019-01-03 10:51:31,077 - DEBUG - musicbot.permissions | In permissions.py::MainThread(11756), line 94 in async_validate: Validating permissions...
[25.3435580730438232] 2019-01-03 10:51:31,077 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 932 in on_ready: Connected: 530334795331272704/JA Music [ALPHA]#1758
[25.3455731868743896] 2019-01-03 10:51:31,079 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 940 in on_ready: Owner:     404689851716206602/Nisar#7302

[25.3465590476989746] 2019-01-03 10:51:31,080 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 943 in on_ready: Guild List:
[25.3465590476989746] 2019-01-03 10:51:31,080 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 947 in on_ready:  - JA GANG
[25.3475580215454102] 2019-01-03 10:51:31,081 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 1003 in on_ready: Not bound to any text channels
[25.3485736846923828] 2019-01-03 10:51:31,082 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 1029 in on_ready: Not autojoining any voice channels
[41.4691083431243896] 2019-01-03 10:51:47,203 - ERROR - musicbot.bot | In bot.py::MainThread(11756), line 903 in on_error: Exception in on_voice_state_update
Traceback (most recent call last):
  File "C:\Users\Junio\AppData\Local\Programs\Python\Python36\lib\site-packages\discord\client.py", line 227, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 2824, in on_voice_state_update
    player = await self.get_player(channel)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 424, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[47.2882452011108398] 2019-01-03 10:51:53,022 - ERROR - musicbot.bot | In bot.py::MainThread(11756), line 903 in on_error: Exception in on_voice_state_update
Traceback (most recent call last):
  File "C:\Users\Junio\AppData\Local\Programs\Python\Python36\lib\site-packages\discord\client.py", line 227, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 2824, in on_voice_state_update
    player = await self.get_player(channel)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 424, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[51.6206605434417725] 2019-01-03 10:51:57,354 - ERROR - musicbot.bot | In bot.py::MainThread(11756), line 903 in on_error: Exception in on_voice_state_update
Traceback (most recent call last):
  File "C:\Users\Junio\AppData\Local\Programs\Python\Python36\lib\site-packages\discord\client.py", line 227, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 2824, in on_voice_state_update
    player = await self.get_player(channel)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 424, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[71.4565448760986328] 2019-01-03 10:52:17,190 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !summon
[71.8636219501495361] 2019-01-03 10:52:17,597 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 697 in deserialize_queue: Deserializing queue for 461264148391657482
[72.7602140903472900] 2019-01-03 10:52:18,494 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 416 in get_player: Created player via deserialization for guild 461264148391657482 with 1 entries
[72.7602140903472900] 2019-01-03 10:52:18,494 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 1886 in cmd_summon: Joining JA GANG/ðŸŽµ Music Room
[72.7652015686035156] 2019-01-03 10:52:18,499 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for audio_cache\youtube-6gjaTkApdBE-Unicorn_Bigger_Surprise_and_Fake_Barbie_LOL_Dolls_Opened_-_Hairgoals_Series_5_Blind_Bags.webm
[72.7652015686035156] 2019-01-03 10:52:18,499 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-6gjaTkApdBE-Unicorn_Bigger_Surprise_and_Fake_Barbie_LOL_Dolls_Opened_-_Hairgoals_Series_5_Blind_Bags.webm
[75.1322515010833740] 2019-01-03 10:52:20,866 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7DBEBE10> using <discord.voice_client.VoiceClient object at 0x000001BE7DB62080>
[75.1372539997100830] 2019-01-03 10:52:20,871 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[75.1382532119750977] 2019-01-03 10:52:20,872 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[82.5626883506774902] 2019-01-03 10:52:28,296 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !skip
[82.7026832103729248] 2019-01-03 10:52:28,436 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 508 in on_player_finished_playing: Running on_player_finished_playing
[82.7036876678466797] 2019-01-03 10:52:28,437 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[82.7396886348724365] 2019-01-03 10:52:28,473 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 504 in on_player_stop: Running on_player_stop
[123.3330204486846924] 2019-01-03 10:53:09,066 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !play geko
[124.6421909332275391] 2019-01-03 10:53:10,376 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 1373 in cmd_play: {'_type': 'url', 'url': 'ytsearch:geko', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'geko', 'webpage_url_basename': 'geko', 'extractor_key': 'Generic'}
[130.4401004314422607] 2019-01-03 10:53:16,174 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[130.4441025257110596] 2019-01-03 10:53:16,178 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 588 in on_player_entry_added: Running on_player_entry_added
[130.4441025257110596] 2019-01-03 10:53:16,178 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[130.4631016254425049] 2019-01-03 10:53:16,197 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=NfqJp3R-9OY
[132.3499145507812500] 2019-01-03 10:53:18,083 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=NfqJp3R-9OY
[132.4491000175476074] 2019-01-03 10:53:18,183 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for audio_cache\youtube-NfqJp3R-9OY-Geko_-_Daily_Duppy_GRM_Daily.webm
[132.4501063823699951] 2019-01-03 10:53:18,184 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-NfqJp3R-9OY-Geko_-_Daily_Duppy_GRM_Daily.webm
[132.5201015472412109] 2019-01-03 10:53:18,254 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7DEA9630> using <discord.voice_client.VoiceClient object at 0x000001BE7DB62080>
[132.5220985412597656] 2019-01-03 10:53:18,256 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[132.5231013298034668] 2019-01-03 10:53:18,257 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[144.1575605869293213] 2019-01-03 10:53:29,891 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !skip
[144.2955582141876221] 2019-01-03 10:53:30,029 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 508 in on_player_finished_playing: Running on_player_finished_playing
[144.2955582141876221] 2019-01-03 10:53:30,029 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[144.3035573959350586] 2019-01-03 10:53:30,037 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 504 in on_player_stop: Running on_player_stop
[576.4891080856323242] 2019-01-03 11:00:42,223 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 502184918080552962/Uncle Taj#0169: !play my little pony ost
[576.7061047554016113] 2019-01-03 11:00:42,440 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 1373 in cmd_play: {'_type': 'url', 'url': 'ytsearch:my little pony ost', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'my little pony ost', 'webpage_url_basename': 'my little pony ost', 'extractor_key': 'Generic'}
[580.1343932151794434] 2019-01-03 11:00:45,868 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[580.1384096145629883] 2019-01-03 11:00:45,872 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 588 in on_player_entry_added: Running on_player_entry_added
[580.1384096145629883] 2019-01-03 11:00:45,872 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[580.1583936214447021] 2019-01-03 11:00:45,892 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=DOmdB7D-pUU
[582.1368453502655029] 2019-01-03 11:00:47,870 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[583.0575909614562988] 2019-01-03 11:00:48,791 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=DOmdB7D-pUU
[583.0645918846130371] 2019-01-03 11:00:48,798 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-DOmdB7D-pUU-My_Little_Pony_-_Friendship_is_Magic_-_Season_1_Opening_1080p.m4a
[583.0715906620025635] 2019-01-03 11:00:48,805 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7DB79160> using <discord.voice_client.VoiceClient object at 0x000001BE7DB62080>
[583.0745918750762939] 2019-01-03 11:00:48,808 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[583.0765962600708008] 2019-01-03 11:00:48,810 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[621.2821660041809082] 2019-01-03 11:01:27,016 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 508 in on_player_finished_playing: Running on_player_finished_playing
[621.2821660041809082] 2019-01-03 11:01:27,016 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[621.2941708564758301] 2019-01-03 11:01:27,028 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 504 in on_player_stop: Running on_player_stop
[6753.3772466182708740] 2019-01-03 12:43:39,111 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 2893 in on_guild_available: Guild "JA GANG" has become available.
[6755.3322134017944336] 2019-01-03 12:43:41,066 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 915 in on_ready: Connection established, ready to go.
[6755.3332126140594482] 2019-01-03 12:43:41,067 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 920 in on_ready: Received additional READY event, may have failed to resume
[8354.2132146358489990] 2019-01-03 13:10:19,947 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 2893 in on_guild_available: Guild "JA GANG" has become available.
[8356.2003023624420166] 2019-01-03 13:10:21,934 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 915 in on_ready: Connection established, ready to go.
[8356.2003023624420166] 2019-01-03 13:10:21,934 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 920 in on_ready: Received additional READY event, may have failed to resume
[10103.0316860675811768] 2019-01-03 13:39:28,765 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !summon
[10103.2324166297912598] 2019-01-03 13:39:28,966 - ERROR - musicbot.bot | In bot.py::MainThread(11756), line 2755 in on_message: Error in summon: CommandError: You are not in a voice channel!
Traceback (most recent call last):
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 2733, in on_message
    response = await handler(**handler_kwargs)
  File "C:\Users\Junio\OneDrive\Documents\Desktop\ja bot\MusicBot\musicbot\bot.py", line 1855, in cmd_summon
    raise exceptions.CommandError(self.str.get('cmd-summon-novc', 'You are not connected to voice. Try joining a voice channel!'))
musicbot.exceptions.CommandError: You are not in a voice channel!
[11877.9362101554870605] 2019-01-03 14:09:03,670 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 435856349855744022/JettersAFC#2386: !play lightning octavian
[11879.1096065044403076] 2019-01-03 14:09:04,843 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 1373 in cmd_play: {'_type': 'url', 'url': 'ytsearch:lightning octavian', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'lightning octavian', 'webpage_url_basename': 'lightning octavian', 'extractor_key': 'Generic'}
[11884.2798576354980469] 2019-01-03 14:09:10,013 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[11884.2838594913482666] 2019-01-03 14:09:10,017 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 588 in on_player_entry_added: Running on_player_entry_added
[11884.2838594913482666] 2019-01-03 14:09:10,017 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[11884.3038606643676758] 2019-01-03 14:09:10,037 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=PgM5n7qU3ds
[11886.2884747982025146] 2019-01-03 14:09:12,022 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[11886.3566491603851318] 2019-01-03 14:09:12,090 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=PgM5n7qU3ds
[11886.3726463317871094] 2019-01-03 14:09:12,106 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-PgM5n7qU3ds-Octavian_-_Lightning_Music_Video_Void_Box.webm
[11890.6277234554290771] 2019-01-03 14:09:16,361 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7DEF7DD8> using <discord.voice_client.VoiceClient object at 0x000001BE7DB62080>
[11890.6297204494476318] 2019-01-03 14:09:16,363 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[11890.6327219009399414] 2019-01-03 14:09:16,366 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[11927.8321139812469482] 2019-01-03 14:09:53,566 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 435856349855744022/JettersAFC#2386: !play octavian lightning
[11928.0151124000549316] 2019-01-03 14:09:53,749 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 1373 in cmd_play: {'_type': 'url', 'url': 'ytsearch:octavian lightning', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'octavian lightning', 'webpage_url_basename': 'octavian lightning', 'extractor_key': 'Generic'}
[11931.8738679885864258] 2019-01-03 14:09:57,607 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[11931.8898665904998779] 2019-01-03 14:09:57,623 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 588 in on_player_entry_added: Running on_player_entry_added
[11931.8898665904998779] 2019-01-03 14:09:57,623 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[11931.9048659801483154] 2019-01-03 14:09:57,638 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 207 in _download: Download cached: https://www.youtube.com/watch?v=PgM5n7qU3ds
[11957.9924876689910889] 2019-01-03 14:10:23,726 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2626 in on_message: 404689851716206602/Nisar#7302: !summon
[11958.3904771804809570] 2019-01-03 14:10:24,124 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 697 in deserialize_queue: Deserializing queue for 461264148391657482
[11958.8315939903259277] 2019-01-03 14:10:24,565 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 416 in get_player: Created player via deserialization for guild 461264148391657482 with 2 entries
[11958.8325958251953125] 2019-01-03 14:10:24,566 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 1886 in cmd_summon: Joining JA GANG/ðŸŽµ Music Room
[11958.9276435375213623] 2019-01-03 14:10:24,661 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for None
[11958.9286401271820068] 2019-01-03 14:10:24,662 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for audio_cache\youtube-PgM5n7qU3ds-Octavian_-_Lightning_Music_Video_Void_Box.webm
[11958.9286401271820068] 2019-01-03 14:10:24,662 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-PgM5n7qU3ds-Octavian_-_Lightning_Music_Video_Void_Box.webm
[11958.9356451034545898] 2019-01-03 14:10:24,669 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7EEF0C50> using <discord.voice_client.VoiceClient object at 0x000001BE7DEB0208>
[11958.9416468143463135] 2019-01-03 14:10:24,675 - INFO - musicbot.entry | In entry.py::MainThread(11756), line 207 in _download: Download cached: https://www.youtube.com/watch?v=PgM5n7qU3ds
[11958.9436442852020264] 2019-01-03 14:10:24,677 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[11958.9436442852020264] 2019-01-03 14:10:24,677 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[12199.7272088527679443] 2019-01-03 14:14:25,461 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 508 in on_player_finished_playing: Running on_player_finished_playing
[12199.7282106876373291] 2019-01-03 14:14:25,462 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[12199.7787508964538574] 2019-01-03 14:14:25,512 - DEBUG - musicbot.entry | In entry.py::MainThread(11756), line 56 in get_ready_future: Created future for audio_cache\youtube-PgM5n7qU3ds-Octavian_-_Lightning_Music_Video_Void_Box.webm
[12199.7797605991363525] 2019-01-03 14:14:25,513 - FFMPEG - musicbot.player | In player.py::MainThread(11756), line 273 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-PgM5n7qU3ds-Octavian_-_Lightning_Music_Video_Void_Box.webm
[12199.7867522239685059] 2019-01-03 14:14:25,520 - DEBUG - musicbot.player | In player.py::MainThread(11756), line 284 in _play: Playing <discord.player.PCMVolumeTransformer object at 0x000001BE7EEF0A58> using <discord.voice_client.VoiceClient object at 0x000001BE7DEB0208>
[12199.7897510528564453] 2019-01-03 14:14:25,523 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 451 in on_player_play: Running on_player_play
[12199.7907471656799316] 2019-01-03 14:14:25,524 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 673 in serialize_queue: Serializing queue for 461264148391657482
[12204.5453920364379883] 2019-01-03 14:14:30,279 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2846 in on_voice_state_update: Pausing in JA GANG/ðŸŽµ Music Room (empty channel)
[12204.5483877658843994] 2019-01-03 14:14:30,282 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 499 in on_player_pause: Running on_player_pause
[12206.1422815322875977] 2019-01-03 14:14:31,876 - INFO - musicbot.bot | In bot.py::MainThread(11756), line 2857 in on_voice_state_update: Unpausing in JA GANG/ðŸŽµ Music Room
[12206.1482799053192139] 2019-01-03 14:14:31,882 - DEBUG - musicbot.bot | In bot.py::MainThread(11756), line 495 in on_player_resume: Running on_player_resume
